common/src/benchmarks.c

```diff
@@ -29,9 +30,22 @@
 #define BUF_SIZE (128 * 1024)  // must be at least 1024
 #endif
 
+#define CSR_ACC_COUNTER     0xcc4
+#define CSR_REFILL_COUNTER  0xcc8
+#define CSR_STALL_COUNTER   0xccc
+
 // Each project should make their own proj_menu.c, which will replace this one.
 
+static void get_csr_metrics(uint32_t *acc, uint32_t *refill, uint32_t *stall) {
```

Comment:
Having these numbers here in the benchmarks is certainly interesting, but the more important place we really need this is for the model evaluation. For example, at the end of `pdti8_classify()` how many cycles did we spend, and how many of these were we stalled on dcache or icache?

Reply:
Good idea, I will add it to all `*_classify` functions.

Reply:
Should be done now.